packages/api/src/controllers/asset.ts

```diff
@@ -55,6 +55,17 @@ function validateAssetMeta(meta: Record<string, string>) {
   }
 }
 
+function cleanAssetResponse(assets: WithID<Asset>[]) {
+  let cleanedAssets = [];
+  for (var i = 0; i < assets.length; i++) {
+    if (assets[i].videoSpec) {
+      delete assets[i].videoSpec.tracks;
+    }
+    cleanedAssets.push(assets[i]);
+  }
+  return cleanedAssets;
```

Comment:
**nit:** Functionally, this could be:
```suggestion
  return assets.map(a => !a.videoSpec ? a : { 
    ...a, 
    videoSpec: {
      ...a.videoSpec,
      tracks: undefined
    }
  })
```
WDYT?

Reply:
Much better!